### PR TITLE
Add ability to Use Fewer Channels w/ Pretrained Model

### PR DESCRIPTION
**Pull request by jamesmcclain**
## Overview

This allows use of mostly-pretrained networks when the number of input channels is fewer than that of the pretrained network.

### Checklist

- [x] Updated `docs/changelog.rst`
- [x] Added `needs-backport` label if PR is bug fix that applies to previous minor release
- [x] Ran scripts/format_code and committed any changes
- [x] Documentation updated if needed
- [x] PR has a name that won't get you publicly shamed for vagueness

### CODE REVIEW

File: rastervision_pytorch_learner/rastervision/pytorch_learner/semantic_segmentation_learner.py [hunk 1]
```diff
@@ -143,10 +143,11 @@ def build_model(self) -> nn.Module:
                 Parallel(old_conv, new_conv),
                 # sum the parallel outputs
                 AddTensors())
+        elif input_channels < old_conv.in_channels:
+            model.backbone.conv1 = nn.Conv2d(
+                in_channels=input_channels, **old_conv_args)
```
Comment (AdeelH) [inline rastervision_pytorch_learner/rastervision/pytorch_learner/semantic_segmentation_learner.py:148]: I would recommend retaining the pretraining by copying over the weights of the first `input_channels` channels of the filters in the `old_conv`.

```python3
new_conv.weight.data[:, :input_channels] = old_conv.weight.data[:, :input_channels]
```

See example here:
https://colab.research.google.com/drive/1MKP6M-TXDoeuxGOumeQhuJFxjyZ-RfI4?usp=sharing

**Review by AdeelH — CHANGES_REQUESTED** [review]

File: rastervision_pytorch_learner/rastervision/pytorch_learner/semantic_segmentation_learner.py (hunk 1 above)
Comment (jamesmcclain) [reply rastervision_pytorch_learner/rastervision/pytorch_learner/semantic_segmentation_learner.py:148]: You are right, I forgot that raster-vision does not deferentially train different parts of the network.

**Review by AdeelH — APPROVED** [review]
